tests/build-gating.fmf

```diff
@@ -0,0 +1,42 @@
+#
+# Snapshot gating tests for all *LLVM* packages in the mix
+#
+# Compatible with various these distributions:
+#
+#   * Fedora
+#
+
+summary: LLVM Tests for snapshot gating
+discover:
+    - name: clang-tests
+      how: fmf
+      url: https://src.fedoraproject.org/tests/clang.git
+      ref: main
+      filter: "tag:-not-in-default"
+    - name: compiler-rt-tests
+      how: fmf
+      url: https://src.fedoraproject.org/tests/compiler-rt.git
+      ref: main
+    - name: libomp-tests
+      how: fmf
+      url: https://src.fedoraproject.org/tests/libomp.git
+      ref: main
+    - name: upstream-llvm-integration-testsuite
+      how: fmf
+      url: https://src.fedoraproject.org/tests/llvm.git
+      ref: main
+      test: integration-test-suite
+    - name: python-lit
+      how: fmf
+      url: https://src.fedoraproject.org/tests/python-lit.git
+      ref: main
+    - name: lld-tests
+      how: fmf
+      url: https://src.fedoraproject.org/tests/lld.git
+      ref: main
```

Comment:
Under this line add `filter: "tag:-spoils-installation & tag:-not-in-default"`. Else you plan will include [this test](https://src.fedoraproject.org/tests/lld/blob/main/f/ld-alternative), which will break your environment.
I would add that filter in each discover step, in case more breaking tests are added in the future.

Reply:
Done in https://github.com/kwk/llvm-daily-fedora-rpms/pull/74/commits/0c5a8d9a882b2d8686751bc16b98ff10018bc342